daemon/algod/api/algod.oas3.yml

```diff
@@ -1623,10 +1623,44 @@
         "description": "Represents a state proof and its corresponding message",
         "properties": {
           "Message": {
-            "description": "The encoded message.",
-            "format": "byte",
-            "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
-            "type": "string"
+            "description": "Represents the message that the state proofs are attesting to.",
```

Comment:
Looks like a little more than a rename, were these fields just missing before?

Reply:
the state proof message could be retrieved only in its msgpack version.